app/src/main/java/de/danoeh/antennapod/fragment/CategoriesFragment.java

```diff
@@ -0,0 +1,59 @@
+package de.danoeh.antennapod.fragment;
+
+import android.support.v4.app.Fragment;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.GridView;
+import java.util.ArrayList;
+import java.util.Arrays;
+import de.danoeh.antennapod.R;
+import de.danoeh.antennapod.activity.MainActivity;
+
+public class CategoriesFragment extends Fragment {
+
+
+    public static final String TAG = "CategoriesFragment";
+    private ArrayAdapter listAdapter;
+    private GridView gridView;
+    private ArrayList<String>categories = new ArrayList<>(Arrays.asList("Arts", "Comedy", "Education", "Kids & family", "Health", "TV & Film", "Music", "News & Politics", "Religion & Spirituality" ,"Science & Medicine", "Sports", "Technology", "Business", "Games & Hobbies", "Society & Culture", "Government & Organizations" ));
```

Comment:
Why don't you use the categories array that you added to the resource file for this purpose?

Reply:
thanks for mentioning this, I forgot about the resource file at some point. Will be corrected.

Reply:
thanks for mentioning this, I forgot about the resource file at some point. Will be corrected.